docs/subsystems/dependencies.md

```diff
@@ -192,7 +192,7 @@ highlighting.  The system is largely managed by the code in
 * **Mypy type checker**.  Because we're using mypy in a strict mode,
   when you add use of a new Python dependency, you usually need to
   either adds stubs to the `stubs/` directory for the library, or edit
-  `mypy.ini` in the root of the Zulip project to configure
+  `pyproject.toml` in the root of the Zulip project to configure
   `ignore_missing_imports` for the new library.  See
```

Comment:
(This has been false since a8f66573f35fef51c8aeb0970c711f0e693c364f, but I’m considering making it true again to prevent bugs like the ones I fixed in #19105 and #19106. Leaving the documentation at status quo for now.)

Reply:
Yeah, I think that makes sense (both leaving documentation as-is and also planning to make it true again)